apps/zui/src/js/initializers/initialize.ts

```diff
@@ -44,11 +44,7 @@ export default async function initialize(
   const api = new ZuiApi()
   const store = await initStore(api, renderer)
   const asyncTasks = initAsyncTasks(renderer)
-  await initGlobals(store)
-  await initLake(store)
-  api.init(store.dispatch, store.getState)
-  initDOM()
-  initIpcListeners(store)
+  initDomainModels({store})
```

Comment:
the Domain Models need to be provisioned with the store variable before we call "initLake". So I re-ordered the steps here.